src/main/webapp/index.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
is this needed for something?

Reply:
I remember that the HTML Presubmit raised an error about index.js when the file didn't exist -- I'll try and see if deleting this file entirely will still cause the presubmit check to fail.

Reply:
I think HTML Presubmit raised an error in the beginning about index.js, which at the time didn't exist. I'll delete the file since it's not needed, and see if this error reappears.